### PR TITLE
✨ Add trackcard associated to tracklist page

### PR DESCRIPTION
**Pull request by fgottesmann**
I add the track card to the track page, which will contain the likebutton when fully merged.

### CODE REVIEW

File: src/app/components/Trackcard(tracklist)/Trackcard-tracklist.module.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.trackcard {
+  border-radius: 100px;
+  border: 1px solid var(--border-color);
+  background-color: var(--background-color);
+  box-shadow: -3px -3px 6px var(--text-color);
+  max-width: 95%;
+  max-height: 12rem;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```
Comment (MaxMuecket) [inline src/app/components/Trackcard(tracklist)/Trackcard-tracklist.module.css:9]:
```suggestion
  grid-template-columns: repeat(7, 1fr);
```

**Review by MaxMuecket — APPROVED** [review]
lgtm ✌️

File: src/app/components/Trackcard(tracklist)/Trackcard-tracklist.module.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.trackcard {
+  border-radius: 100px;
+  border: 1px solid var(--border-color);
+  background-color: var(--background-color);
+  box-shadow: -3px -3px 6px var(--text-color);
```
Comment (phlgr) [inline src/app/components/Trackcard(tracklist)/Trackcard-tracklist.module.css:5]: Please use relative units like em or rem

File: src/app/components/Trackcard(tracklist)/Trackcard-tracklist.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './Trackcard-tracklist.module.css';
+
+export type Props = {
```
Comment (phlgr) [inline src/app/components/Trackcard(tracklist)/Trackcard-tracklist.tsx:4]:
```suggestion
export type TrackcardProps = {
```

**Pull request merged**